Automattic/pocket-casts-ios
# [Winback] Filter view based on client

| 📘 Part of: #2435
|:---:|

Fixes #2507

Shows the new Cancel Subscription screen only if the subscription platform is iOS. For all other platforms we will display the current web view.

| 1 | 2 | 3 |
| -------- | ------- | ------- |
| ![RocketSim_Screenshot_iPhone_16_6 1_2024-12-17_14 32 23](https://github.com/user-attachments/assets/50a8883f-f52a-44db-880b-6de8efe5de14) | ![RocketSim_Screenshot_iPhone_16_6 1_2024-12-17_14 34 47](https://github.com/user-attachments/assets/ebd496b7-35cf-427a-b8e9-3f33e4da8648) | ![RocketSim_Screenshot_iPhone_16_6 1_2024-12-17_14 34 51](https://github.com/user-attachments/assets/5c06c880-d9bc-48a3-988d-d55bbe790c17) |

## To test

- CI must be 🟢 
- Make sure you have what `winback` FF on
- Login with a premium user where the subscription is from another platform (Android/Web)
- Go to Account -> Cancel Subscription
- You should see the legacy screen
- Tapping on

## Checklist
